tizen-api/meson.build

```diff
@@ -15,8 +15,6 @@ if meson.project_name() != 'nnstreamer'
 
   cc = meson.get_compiler('c')
   cxx = meson.get_compiler('cpp')
-
-  gst_api_verision = '1.0'
```

Comment:
@jaeyun-jung , Will not we use the version information of API after this PR? 
Or We may remove this variable because this variable is not used?

Reply:
We don't use it.